slatedb/src/compactor_executor.rs

```diff
@@ -138,30 +138,26 @@ impl TokioCompactionExecutorInner {
         compaction: &'a CompactionJob,
     ) -> Result<RetentionIterator<MergeIterator<'a>>, SlateDBError> {
         let sst_iter_options = SstIteratorOptions {
-            max_fetch_tasks: 4,
-            blocks_to_fetch: 256,
+            max_fetch_tasks: 16,
```

Comment:
I wonder if making this a function of the number of available CPUs could improve performance depending on the machine you’re running.

Reply:
Yeah. It should probably not block these changes anyways. It could be an improvement over this.